action: add benchmark on schedule

Desiki-high opened the pull request:
## Add the benchmark workflow
1. We will cache each benchmark result and compare with the last result.
2. See [docs.github](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy), gitHub will remove any cache entries that have not been accessed in over 7 days. We schedule benchmark on every Monday and Wednesday. You can also do this manually.

## Add benchmark-schedule mode on benchmark summary
You can see the result in [1](https://github.com/dragonflyoss/image-service/actions/runs/4851221114/attempts/1#summary-13144939774), [2](https://github.com/dragonflyoss/image-service/actions/runs/4851221114/attempts/3#summary-13145178832), [3](https://github.com/dragonflyoss/image-service/actions/runs/4851221114/attempts/4#summary-13145339659), [4](https://github.com/dragonflyoss/image-service/actions/runs/4851221114/attempts/4#summary-13145339659).

> The pull request trigger for dev was deleted in [compare](https://github.com/dragonflyoss/image-service/compare/34c2d457ad31e7c70296bcabc6e53105c8ef4fac..83fbf80b335570b4bca99faeccdc43fee7626a25).
> Next work: add more images in benchmark.

## Ref:https://github.com/dragonflyoss/image-service/issues/1220

hsiangkao commented:
switch into daily or every two days?

Desiki-high commented:
> switch into daily or every two days?



> switch into daily or every two days?



> switch into daily or every two days?

Is that too frequent? We want to compare this benchmark with the last time benchmark.

hsiangkao commented:
> > switch into daily or every two days?
> 
> Is that too frequent? We want to compare this benchmark with the last time benchmark.

I'm afraid it's hard to find the real commit which causes regression with a per-week benchmark test.

Desiki-high commented:
> > > switch into daily or every two days?
> > 
> > 
> > Is that too frequent? We want to compare this benchmark with the last time benchmark.
> 
> I'm afraid it's hard to find the real commit which causes regression with a per-week benchmark test.

You are right. I will switch to every two days.

Desiki-high committed:
action: add benchmark on schedule

we will run benchmark twice per-week, and compare result with the last by cache.

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

Desiki-high committed:
misc: supprot benchmark on schedule

supprot the benchmark on schedule by new mode in benchmark_summary.

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

imeoer reviewed (approved):
LGTM!
